hal/src/generated/main/java/edu/wpi/first/hal/FRCNetComm.java

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+/*
+ * Autogenerated file! Do not manually edit this file.
+ */
```

Comment:
We should change this to match the message we put into the other generated files in recent PRs (which includes the script name).